### ISSUE TITLE
Embedded platform views don't support Voice Access

### ISSUE THREAD

**Issue by mklim** [flutter/flutter#40913]
Embedded platform views appear to lack basic functionality with Voice Access. I was able to open the hamburger menu inside of our webview app, but nothing else in the embedded UI had controls.

This is likely blocked on #40912. 

/cc @amirh @goderbauer
